Comments/Models/Context.cs

```diff
@@ -965,5 +965,21 @@ public bool AreAnswersForThisOrganisation(List<int> answerIds, int organisationI
 				.Any(c => c.OrganisationUser.OrganisationAuthorisation.OrganisationId.Equals(organisationId));
 
 		}
-	}
+
+        public List<string> GetEmailAddressForCommentsAndAnswers(CommentsAndQuestions commentsAndQuestions)
+        {
+            var commentIds = commentsAndQuestions.Comments.Select(c => c.CommentId).ToList();
+            var answerIds = commentsAndQuestions.Questions.SelectMany(q => q.Answers).Select(a => a.AnswerId).ToList();
+
+
+            var emailAddresses = OrganisationUser
+                .Where(o => Comment.Where(c => commentIds.Contains(c.CommentId)).Select(c => c.OrganisationUserId).Contains(o.OrganisationUserId)
```

Comment:
regarding the use of Comment and Answer subqueries here...

Is is all handled by the database, or is it brought back to the application server?

Also, does it requery the comment and answer tables repeatedly for each organisation user?

If so, i think it needs changing.

Reply:
Need to chat to you about this